drivers/hmis/app/models/hmis/hud/validators/base_validator.rb

```diff
@@ -44,6 +44,10 @@ def self.before_entry_message(entry_date)
     "cannot be before entry date (#{entry_date.strftime('%m/%d/%Y')})"
   end
 
+  def self.on_entry_message(entry_date)
```

Comment:
(though, I'm sure you saw, it looks like this introduces some other test failures that are probably easily fixed)